### PR TITLE
Fix file name and add a few details

### PR DESCRIPTION
**Pull request by jpetazzo**
I tried this yesterday and found that the instructions had to be adjusted slightly.

One of the YAML file names was incorrect, and the namespace has to be passed
as a command line argument to the controller when deploying it.

I also added a few steps to clarify what's required (for somebody who
has never used warp and perhaps never used cloudflare either).

If you don't like the way I rephrased the instructions, feel free to drop this PR
and incorporate just the changes that you like :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,159 @@
...
 each namespace.
 
-#### Cloudflare certificate
 
-The _cert.pem_ file must be available to the controller as a secret,
-and must be configured in the cluster before the controller can start.
+#### Pre-requirements
+
+To use this, you need:
+- a Cloudflare account (free tier is OK)
```
Comment (elithrar) [inline README.md]: Link to https://www.cloudflare.com/ whilst we're at it?

File: README.md [hunk 2]
```diff
@@ -1,60 +1,159 @@
...
+- a zone (domain name) in your Cloudflare account 
+- a Kubernetes cluster
```
Comment (elithrar) [inline README.md]: Link to https://kubernetes.io/

File: README.md [hunk 3]
```diff
@@ -1,60 +1,159 @@
...
+
+The deployment instructions below use a few YAML files that
+will be used with `kubectl` to create the appropriate resources
+in Kubernetes. You might want to checkout that repository
+to have them handy!
+
+
+#### Get warp credentials
```
Comment (elithrar) [inline README.md]: s/warp/Warp (proper noun)

File: README.md [hunk 4]
```diff
@@ -1,60 +1,159 @@
...
+
+The first step is to obtain the credentials that will
+be used by the controller to authenticate with Cloudflare.
 
+First, install the warp client.
```
Comment (elithrar) [inline README.md]: Same here - s/warp/Warp (proper noun)

Comment (jpetazzo) [reply README.md]: Good call; I didn't know if you referred to it as "warp" or "Warp" canonically. I've updated it to "Warp" everywhere. 👍🏻

**Comment by elithrar** [comment]
Thanks @jpetazzo!

@danigrant or @cbranch - can you also review / approve as PM / lead for Warp?

**Comment by MaxMilton** [comment]
Thanks @jpetazzo , this actually helped me to get the service running correctly!

**Pull request merged**